### ISSUE TITLE
Component: Dropdown overlay aria-hidden-focus

### ISSUE THREAD

**Issue by Draganlazarev90** [primefaces/primeng#14985]
### Describe the bug

![image](https://github.com/primefaces/primeng/assets/23295567/e2ed5d43-2633-429b-b157-410024494d33)
WCAG 2.1 violation aria-hidden-focus

### Environment

primeng 17.10

### Angular version

17.0.5

### PrimeNG version

Primeng 17.11

### Build / Runtime

Angular CLI App

### Language

TypeScript

### Node version (for AoT issues node --version)

18.18.0

**Comment by TaneliTuomola** [comment]
There is a same issue with TreeSelect and MultiSelect: Two empty span element between item wrapper div (p-treeselect-items-wrapper &p-multiselect-items-wrapper) with attribute aria-hidden="true" contains focusable elements. Focusable content should have tabindex="-1" or be removed from the DOM.
